src/services/PhotoService.ts

```diff
@@ -3,18 +3,29 @@ import { IService } from "../types/IService";
 import PhotoRepository from "../repositories/PhotoRepository";
 import Photo from "../entities/Photo";
 import PhotoCreateValidator from "../validators/Photo/PhotoCreateValidator";
+// import PhotoValidator from "../validators/Photo/PhotoValidator";
```

Comment:
If we are not using it, then remove it, let's not leave unnecessary commented code!